modules/caddyhttp/templates/tplcontext.go

```diff
@@ -438,6 +439,30 @@ func (c TemplateContext) funcHumanize(formatType, data string) (string, error) {
 	return "", fmt.Errorf("no know function was given")
 }
 
+// ProxyResponse accesses the response from a reverse proxy.
+// It can only be used inside a handle_response route.
+// EXPERIMENTAL: This API is subject to change or removal.
+func (c TemplateContext) ProxyResponse() (proxyResponse, error) {
+	resp, err := reverseproxy.Response(c.Req)
```

Comment:
FYI there's a lint warning here... but I think we can ignore it `response body must be closed (bodyclose)` because the body  actually does get closed here 
https://github.com/caddyserver/caddy/blob/f259ed52bb3764ce4fd5d88f1712cb43247c2639/modules/caddyhttp/reverseproxy/reverseproxy.go#L846

```suggestion
	//nolint:bodyclose
	resp, err := reverseproxy.Response(c.Req)
```

https://golangci-lint.run/usage/false-positives/#nolint-directive